assets/pages/contributions-landing/ContributionState.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { usStates, caStates } from 'helpers/internationalisation/country';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+// ----- Types ----- //
+type PropTypes = {
+  countryGroupId: CountryGroupId,
+  selectedState: string,
+};
+
+// ----- Render ----- //
+
+const renderState = (selectedState: string) => (state: [string, string]) => (
```

Comment:
could be nice to use array destructuring here to make it more readable, e.g. `([stateCode, stateName]: [string, string])`

Reply:
Much nicer indeed!